gitwrapperlib/gitwrapperlib.py

```diff
@@ -159,6 +165,25 @@ def get_current_branch(self):
                      if branch.startswith('*')),
                     None)
 
+    def get_default_branch(self):
+        """Returns the remote default branch."""
+        branch = None
+
+        try:
+            remote = str(self._git.remote('show', 'origin'))
+            match = re.search(r'HEAD branch: (\S+)', remote)
+            if match:
+                branch = match.group(1)
+                if branch == '(unknown)':
+                    branch = None
+        except sh.ErrorReturnCode:
+            raise RemoteNotFound
```

Comment:
Isn't this a little generic? Are we sure that on every error return code it would be a remote not found? At least we need to also print the message so users can debug further.